controls/bsi_app_4_4.yml

```diff
@@ -313,51 +313,128 @@ controls:
     levels:
     - elevated
     description: >-
-      Nodes SHOULD send a cryptographically secured (and, if possible, TPM-verified) status
-      message to the control plane. The control plane SHOULD ONLY accept nodes into a cluster
+      (1) Nodes SHOULD send a cryptographically secured (and, if possible, TPM-verified) status
+      message to the control plane. (2) The control plane SHOULD ONLY accept nodes into a cluster
       that have successfully proven their integrity.
     notes: >-
-      TBD
-    status: pending
-    rules: []
+      OpenShift Nodes are using Red Hat CoreOS (RHCOS) by default, an immutable operating system. 
+      While RHEL is also supported for Compute Nodes, RHCOS is mandatory for Control Plane Nodes and 
+      recommended for all nodes. The correct version and configuration of RHCOS is verified 
+      cryptographically with the desired state, that is managed by the Control Plane using MachineConfigs. 
+      Any manual change on managed files is overwritten to ensure the desired state. Therefore, the 
+      control is mostly inheretly met when using CoreOS for all nodes.
+      
+      Section 1: OpenShift uses an internal Certificate Authority (CA). The nodes (kubelet to API server
+      and MachineConfig daemon to MachineConfi server) are communicating using node-specific certificates,
+      signed by this CA. Correct permissions of relevant files and secure TLS configuration are verified
+      using the referenced rules.
+
+      Section 2: Using the Red Hat File Integrity Operator, all files on the RHCOS nodes can be
+      cryptographically checked for integrity using Advanced Intrusion Detection Environment (AIDE).
+    status: automated

```

Comment:
I would argue this is only partial.

While the status is send over tls connections, we do not make use of TPM afaik. IMHO this requirement from BSI is looking for stuff like https://keylime.dev/ which is currently not build in.

While the rules mentioned work towards the goal of the requirement they are not fully addressing them.

Reply:
From that argumentation it indeed makes sense to go to `partial`. My interpretation for `automated` was based on the phrase `if possible`, which I read as `optional`.

I will add a comment on TPM in the notes and switcht to partial